qsiprep/workflows/anatomical/volume.py

```diff
@@ -1214,7 +1218,7 @@ def init_anat_derivatives_wf() -> Workflow:
             to="T1w",
             mode="image",
             suffix="xfm",
-            **{"from": config.workflow.anatomical_template},
+            **{"from": anatomical_template},
```

Comment:
@mattcieslak I think the only output change will be to these transform files. Instead of `from-MNIInfant_to-T1w` and vice versa, it'll be `from-MNIInfant+<cohort>_to-T1w`.

Reply:
That will have implications for QSIRecon though.

Reply:
We're updating that part of qsirecon right now anyways, so this should be ok